backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -165,6 +184,117 @@ class SchedulingService implements ISchedulingService {
     return schedulingDtos;
   }
 
+  async sendEmailVerificationAfterSchedulingADonation(
+    email: string,
+    schedule: SchedulingDTO,
+  ): Promise<void> {
+    if (!this.emailService) {
+      const errorMessage =
+        "Attempted to call sendEmailVerificationAfterSchedulingADonation but this instance of AuthService does not have an EmailService instance";
+      Logger.error(errorMessage);
+      throw new Error(errorMessage);
+    }
+
+    try {
+      const accountName = await this.userService.getUserByEmail(email);
+      // Proposed drop off info
+      const days = [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+      ];
+      const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+      ];
+      const { startTime } = schedule;
+      const { endTime } = schedule;
+
+      const donationDay: string = days[startTime.getDay()];
+      const donationMonth: string = months[startTime.getMonth()];
+      const startTimePeriod: string = startTime.getHours() < 11 ? "am" : "pm";
+      const endTimePeriod: string = endTime.getHours() < 11 ? "am" : "pm";
+
+      // dropoff time interval string
+      const startTimeString = `${
+        startTime.getHours() === 0 ? 12 : startTime.getHours() % 12
+      }:${startTime.getMinutes()}${startTimePeriod}`;
+      const endTimeString = `${
+        endTime.getHours() === 0 ? 12 : endTime.getHours() % 12
+      }:${endTime.getMinutes()}${endTimePeriod}`;
```

Comment:
this isn't converting properly for me, the getHours is not going from UTC to the correct local time

also, syntax wise, it might be easier to leverage dayjs or moment like on the frontend and use the `get12HTimeString` from the `frontend/Scheduling` ?